### DISCUSSION TITLE
Schema for Company Profile

### DISCUSSION THREAD

**Discussion by heiheihang** [LalaCuteGroup/lala-referral#21] [Ideas]
- `Company Name` (String - **Partition Key**)
- `Location` (String - **Sort Key**)
-  `Leetcode Threshold` (Number)
![Screenshot 2022-07-11 at 10 08 18 PM](https://user-images.githubusercontent.com/24492138/178393233-0d23c780-5ca9-4083-b83d-67a085c43416.png)

**Comment by jonowo** [comment]
Company ID (Number) was mentioned in #17. I think it should be a field (and partition key) here.

How will we associate company profile / job post with referrer? Should we consider the case where there are multiple referrers from the same company?

ඞඞඞ

**Reply by heiheihang** [reply]
I made a mistake in [#17](https://github.com/LalaCuteGroup/lala-referral/discussions/17). I think `Company Name` is more suitable to be a *Partition Key* as in DynamoDB there is no idea of `ID`. 

When we have multiple referrers from the same company, ideally the TG Bot can select one of the active referrers from the *pool* of referrers when an applicant applies to a company. I should mention that somewhere in my design doc.
